.github/workflows/main.yml

```diff
@@ -12,10 +12,17 @@ jobs:
         with:
           node-version: 12.x
       - name: Install Oasis
-        run: curl --proto '=https' --tlsv1.2 -sSL https://get.oasis.dev | python - --speedrun
+        run: curl --proto '=https' --tlsv1.2 -sSL https://get.oasis.dev | python - --speedrun --toolchain unstable
+      - name: Lint
+        run: |
+          pushd app
+          npm install
+          npm run-script lint
+          popd
       - name: Build and test service
         run: |
           export PATH=~/.local/bin:-~/.cargo/bin:$PATH
           export LD_LIBRARY_PATH=$(rustc --print sysroot)/lib:$LD_LIBRARY_PATH
           oasis build
+          oasis chain >& oasis.log &
```

Comment:
it'd be more useful to dump this to stdout unless you have a way to get this from circleci